modules/ROOT/pages/upgrade.adoc

```diff
@@ -71,17 +75,17 @@ the cluster is updating in background
 $ sudo gravity enter
 $ kubectl get pods
 ----
-+
-* Login to the Ops Center at the following URL:
+
+. Log in to the Ops Center at the following URL:
 +
 ----
 https://{platform-host}:9500
 ----
 +
 [WARNING]
-Before accessing Anypoint Platform PCE, ensure that you configure your NSF servers described in the following step. Failure to configure NFS before accessing the platform can result in data loss.
+If you are upgrading from Anypoint Platform PCE 1.7.x to 2.0.x, ensure that you configure your NSF servers described in the following step before accessing Anypoint Platform PCE. In this case, failure to configure NFS before accessing the platform can result in data loss.
```

Comment:
This does not belong in the task steps. 
The previous warnings AND the instructions that follow make clear what the user needs to do. 
It is not doc's job to account for corner-case users who can't follow instructions, if that is what is going on here.
If so, I would consider pushing back a bit if your dev reviewers are pressuring you for these constant warnings and notes; it sounds like the product has some flaws or lack of intuitive UI that the documentation cannot be sacrificed in an effort to rectify.